### DISCUSSION TITLE
Resource's for seaweed cluster for 20TB

### DISCUSSION THREAD

**Discussion by therus000** [seaweedfs/seaweedfs#4921] [Q&A]
Good day
i want to up seaweed cluster for 20TB
what the best practice for resource 
how many node i should make for this cluster and how many CPU and Ram for each node
anyone can help for calculate the resource?

**Comment by kmlebedev** [comment]
> Good day i want to up seaweed cluster for 20TB what the best practice for resource how many node i should make for this cluster and how many CPU and Ram for each node anyone can help for calculate the resource?

This greatly depends on the number of GET/PUT requests. Since this has a linear effect on CPU and RAM consumption.
You can run a weed server locally and limit the number of CPUs via export `GOMAXPROCS=1`

Next, run a [benchmark](https://github.com/seaweedfs/seaweedfs#run-warp-and-launch-a-mixed-benchmark) that is as close as possible to your actual peak load per server.
And get CPU and RAM consumption
